dataproviderdemo/src/main/java/ch/srgssr/dataprovider/demo/MainActivity.kt

```diff
@@ -29,9 +32,11 @@ class MainActivity : AppCompatActivity() {
             emit(ilService.getTvLatestEpisodes(Bu.RTS))
         }
 
-        lifecycleScope.launchWhenResumed {
-            flowData.collectLatest {
-                Log.d("Coucou", "${it.list}")
+        lifecycleScope.launch {
+            lifecycle.repeatOnLifecycle(Lifecycle.State.RESUMED) {
+                flowData.collectLatest {
+                    Log.d("Coucou", "${it.list}")
```

Comment:
Coucou à toi aussi @StaehliJ 😁

Reply:
Shouldn't any Android linter be configured to report errors when a _Coucou_ is somehow detected? 😜

Reply:
Bonjorno Loïc ;)

Reply:
> Shouldn't any Android linter be configured to report errors when a _Coucou_ is somehow detected? 😜

Or rather warn if "Coucou" is not used 😛

Reply:
I let you add a UI to display the list instead of the "coucou" log :P

Reply:
I can send a PR that replaces the default `Hello World!` text with `Loading...` and then the names of the loaded episodes (or the received error message).